### PR TITLE
Entity model related

### PR DESCRIPTION
**Pull request by fewizz**

### CODE REVIEW

**Review by asiekierka — APPROVED** [review]

File: mappings/net/minecraft/client/gui/widget/RealmsButtonWidget.mapping [hunk 1]
```diff
@@ -12,7 +12,6 @@ CLASS ctl net/minecraft/client/gui/widget/RealmsButtonWidget
 		ARG 4 y
 		ARG 5 text
 		ARG 6 width
-	METHOD a (Ljava/lang/String;)V
```
Comment (ChloeDawn) [inline mappings/net/minecraft/client/gui/widget/RealmsButtonWidget.mapping:15]: Unrelated change

File: mappings/net/minecraft/resource/PackResourceManager.mapping [hunk 2]
```diff
@@ -8,7 +8,6 @@ CLASS wn net/minecraft/resource/PackResourceManager
 	FIELD c type Lvx;
 	METHOD a open (Lqh;Lvw;)Ljava/io/InputStream;
 	METHOD a add (Lvw;)V
-	METHOD b (Lqh;)Z
```
Comment (ChloeDawn) [inline mappings/net/minecraft/resource/PackResourceManager.mapping:11]: Unrelated change

**Review by ChloeDawn — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/client/render/entity/model/VillagerEntityModel.mapping [hunk 3]
```diff
@@ -1,3 +1,4 @@
 CLASS dbx net/minecraft/client/render/entity/model/VillagerEntityModel
 	FIELD a head Ldcj;
 	FIELD l nose Ldcj;
+	METHOD a ()Ldcj;
```
Comment (ChloeDawn) [inline mappings/net/minecraft/client/render/entity/model/VillagerEntityModel.mapping]: Missing mapping?

File: mappings/net/minecraft/client/render/entity/model/EvilVillagerEntityModel.mapping [hunk 4]
```diff
@@ -1 +1,2 @@
 CLASS dae net/minecraft/client/render/entity/model/EvilVillagerEntityModel
+	METHOD a ()Ldcj;
```
Comment (ChloeDawn) [inline mappings/net/minecraft/client/render/entity/model/EvilVillagerEntityModel.mapping]: Missing mapping?

File: mappings/net/minecraft/client/render/entity/model/BipedEntityModel.mapping [hunk 5]
```diff
@@ -13,7 +13,17 @@ CLASS dad net/minecraft/client/render/entity/model/BipedEntityModel
...
+	METHOD a lerpAngle (FFF)F
+		ARG 1 a
+		ARG 2 b
+		ARG 3 t
+	METHOD a (FLaht;)V
+		ARG 1 arg1
 	METHOD a getArm (Laht;)Ldcj;
 	METHOD a getPreferedHand (Lahu;)Laht;
 	METHOD a setAttributes (Ldad;)V
+	METHOD b (Lahl;FFFFFF)V
```
Comment (ChloeDawn) [inline mappings/net/minecraft/client/render/entity/model/BipedEntityModel.mapping:26]: Missing mapping?

File: mappings/net/minecraft/client/render/entity/model/BipedEntityModel.mapping [hunk 6]
```diff
@@ -13,7 +13,17 @@ CLASS dad net/minecraft/client/render/entity/model/BipedEntityModel
 	METHOD <init> (FFII)V
 		ARG 1 scale
 		ARG 3 textureWidth
+		ARG 4 textureHeight
+	METHOD a ()Ldcj;
+	METHOD a lerpAngle (FFF)F
+		ARG 1 a
+		ARG 2 b
+		ARG 3 t
+	METHOD a (FLaht;)V
```
Comment (ChloeDawn) [inline mappings/net/minecraft/client/render/entity/model/BipedEntityModel.mapping:21]: Missing mapping?

File: mappings/net/minecraft/client/render/entity/model/BipedEntityModel.mapping [hunk 7]
```diff
@@ -13,7 +13,17 @@ CLASS dad net/minecraft/client/render/entity/model/BipedEntityModel
 	METHOD <init> (FFII)V
 		ARG 1 scale
 		ARG 3 textureWidth
+		ARG 4 textureHeight
+	METHOD a ()Ldcj;
```
Comment (ChloeDawn) [inline mappings/net/minecraft/client/render/entity/model/BipedEntityModel.mapping]: Missing mapping?

File: mappings/net/minecraft/client/render/entity/model/BipedEntityModel.mapping (hunk 6 above)
Comment (asiekierka) [reply mappings/net/minecraft/client/render/entity/model/BipedEntityModel.mapping:21]: There's a magic method to get rid of these artifacts (@gegy1000 please fix):

1. gradle yarn
2. save
3. close enigma

Comment (Gegy) [reply mappings/net/minecraft/client/render/entity/model/BipedEntityModel.mapping:21]: This issue is hopefully resolved in https://github.com/FabricMC/Enigma/commit/b34305a235cdaaeadecad874931f2f2b62971711

Comment (fewizz) [reply mappings/net/minecraft/client/render/entity/model/BipedEntityModel.mapping:21]: After resaving:
screenshot from 2019-02-16 19-42-52
Or i should fix that manually?

Comment (asiekierka) [reply mappings/net/minecraft/client/render/entity/model/BipedEntityModel.mapping:21]: No! Not manually!

**Comment by asiekierka** [comment]
Why? Our changes don't fully converge with each other. Keep it open - I'll handle the merging.
